deployment/frontend/src/pages/topics/[topicName].tsx

```diff
@@ -9,38 +9,52 @@ import { useState } from 'react'
 import { SearchInput } from '@/schema/search.schema'
 import { api } from '@/utils/api'
 import { NextSeo } from 'next-seo'
-
-const links = [
-    { label: 'Topics', url: '/topics', current: false },
-    { label: 'Topics 1', url: '/topics/test', current: true },
-]
+import { useRouter } from 'next/router'
+import { GroupTree } from '@/schema/ckan.schema'
+import DatasetTopic from '@/components/topics/DatasetTopic'
+import { getServerAuthSession } from '@/server/auth'
+import Spinner from '@/components/_shared/Spinner'
+import GroupBreadcrumb from '@/components/team/GroupBreadcrumb'
 
 export default function TopicPage() {
```

Comment:
I dont think you need to necessarly use TRPC here(Unless its easier for you) having said that you should still call use getServersideProps, either by providing an initialData like here https://github.com/wri/wri-odp/blob/dev/deployment/frontend/src/pages/datasets/%5BdatasetName%5D.tsx#L47  or doing prefetching like here https://github.com/wri/wri-odp/blob/dev/deployment/frontend/src/pages/dashboard/datasets/%5BdatasetName%5D/edit.tsx#L36 https://trpc.io/docs/v9/ssg-helpers